tests/.gitignore

```diff
@@ -0,0 +1,162 @@
+output/*
```

Comment:
I'm not sure that `tests` is the right place for your tool.
Could you please consider moving it to the `tools` folder?